### DISCUSSION TITLE
Minor enhancement - js scripting in email content - no scroll bar

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#11046] [Help] [answered]
Hi,

I am doing some formatting of content for the body of an email using javascript in automate.

It appears once you fill the initial editor box size, no scroll bar appears. To move up or down you have to use the keyboard up/down arrows. It would be beneficial if you could scroll up or down.

![image](https://github.com/Budibase/budibase/assets/83434591/36b174b4-3ba6-49a1-8665-e1e48a559542)

Thanks
Ben

**Comment by ConorWebb96** [comment, marked as answer]
Hey @benaunt,

There is an issue created for this! It seems that there is a fix in place for this. The issue created can be found here; https://github.com/Budibase/budibase/issues/10990

I hope this helps!
